### ISSUE TITLE
syntax error after updating to iTerm2 3.0.0

### ISSUE THREAD

**Issue by dvinegla** [mklement0/ttab#4]
```
ttab
77:85: syntax error: Expected “,” but found identifier. (-2741)
```

**Comment by mklement0** [comment]
@dvinegla Thanks for letting me know. Unfortunately, https://www.iterm2.com/version3.html states (emphasis mine):

> Applescript support has been updated, but it's **not backwards compatible**.

I'll take a look.

**Comment by mklement0** [comment]
@dvinegla: I've just released **v0.3.1**, which now supports iTerm2 v3.

As before, I haven't gotten around to writing automated tests - do let me know if you find problems.
